src/TypeDefinition.js

```diff
@@ -140,16 +147,17 @@ export type PossiblyDeprecatedNavigationAction =
  * The state for the root navigator of our app represents the whole navigation
  * state for the whole app.
  */
+
+// TODO: rename NavigationState to NavigationParentState
```

Comment:
do we want to address renaming this before landing this PR or is something else blocking it?

Reply:
no thanks!